scripts/fetch_filaments.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
```

Comment:
Unnecessary since HA doesn't support old python versions.

Reply:
![image](https://github.com/user-attachments/assets/fe2bc620-b24d-44a8-a800-c2175a3b21bb)

I copied it from the other files which use it